doc/man3/OSSL_PARAM.pod

```diff
@@ -197,8 +197,8 @@ enough set of data, that call should succeed.
 =item *
 
 A I<responder> must never change the fields of an C<OSSL_PARAM>, it
-may only change the contents of the memory that C<data> and
-C<return_size> point at.
+may only change the contents of the memory that C<data> points at and
+the C<return_size>.
```

Comment:
This needs a rephrase, 'cause "the responder must never change the fields... may change return_size" is a bit confusing.

Reply:
Reworded this one.